app/queries/get_linked.rb

```diff
@@ -13,10 +13,8 @@ def call
       validate_fields!
 
       content_ids = Link
-        .where(target_content_id:)
-        .where(link_type:)
-        .joins(:link_set)
-        .pluck(:content_id)
+                      .where(target_content_id:, link_type:)
+                      .pluck(:link_set_content_id)
 
       editions = Edition.with_document.where("documents.content_id": content_ids)
```

Comment:
I think I could have done:

```
Edition.joins(document: :reverse_links).where(links: { target_content_id:, link_type: })
```

... and saved a separate query for the content_ids, but I think the code is easier to understand and plenty performant without that.

Reply:
It also seems a bit odd to me that this query will return all editions, regardless of their state / content_store. But 🤷🏻  I guess it works.